OpenRA.Mods.Common/Traits/Buildings/FootprintUtils.cs

```diff
@@ -30,6 +30,10 @@ public static IEnumerable<CPos> Tiles(Ruleset rules, string name, BuildingInfo b
 				footprint = footprint.Concat(new char[dim.X]);
 			}
 
+			// Don't exclude '_' for FrozenUnderFog.
+			if (includePassable)
+				return TilesWhere(name, dim, footprint.ToArray(), a => !char.IsWhiteSpace(a)).Select(t => t + topLeft);
```

Comment:
You shouldn't need the `IsWhiteSpace` check by that point, you can simply set a `true` function. Whitespaces are removed in L24 afterall.